yt_dlp/extractor/dropout.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+import requests
```

Comment:
remove dependency and use `self._download_json`, `self._download_webpage` etc.
Look at other extractors for example